### PR TITLE
docs(landing-page): brand NEW landing page

### PR DESCRIPTION
**Pull request by SoonIter**
### Tasks

- [x] kickoff: static XL size landing page
- [x] i18n
- [x]  click event
- [x] responsive layout: L -> M -> S
- [x] dark mode
- [x] animation (I'll ask for animation files in this step I think)
- [ ] extract the shared components (e.g Tool Stack) to other websites (Rsbuild,Rsdoctor, etc)

```
// XL 1440
// L 1280
// M 834
// S 393
```

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Comment by SoonIter** [comment]
"extract the shared components"
I think rspress theme is a better choice

File: website/theme/components/Landingpage/Benchmark/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  padding: 0px 144px 32px 144px;
+  flex-direction: column;
+  align-items: center;
+  gap: 32px;
+
+  .title {
+    color: var(--landingpage-title-color);
+    text-align: center;
+    /* Style 10 */
+    font-family: 'Open Sans';
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/Benchmark/index.module.scss:12]: No need to set `Open Sans`, it's the same as default font-family

File: website/theme/components/Landingpage/Benchmark/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+// Benchmark data for different cases
+// Unit: second
+// From: https://github.com/rspack-contrib/performance-compare
+const BENCHMARK_DATA: BenchmarkData = {
+  rspack: {
+    label: 'Rspack',
+    metrics: [
+      {
+        time: 0.49,
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/Benchmark/index.tsx:23]: Use the latest metrics from the main branch

File: website/theme/components/Landingpage/Background.compressed.png
Comment (chenjiahan) [inline website/theme/components/Landingpage/Background.compressed.png:1]: Images can be moved to https://github.com/rspack-contrib/rstack-design-resources

File: website/theme/i18n/zhCN.ts [hunk 3]
```diff
@@ -1,40 +1,76 @@
...
-  ecosystem: '生态',
-  community: '社区',
+  // hero
+  heroSlogan: '基于 Rust 的高性能 Web 打包工具',
+  heroSubSlogan: '使用兼容 API 无缝替换 webpack',
+  getStarted: '快速开始',
+  learnMore: '深入了解',
+
+  // whyRspack
+  whyRspack: '为什么使用 Rspack？',
```
Comment (chenjiahan) [inline website/theme/i18n/zhCN.ts:11]: 这里用 "什么是" 可能更合适，比较简短
```suggestion
  whyRspack: '什么是 Rspack？',
```

File: website/theme/components/Landingpage/Background.compressed.png
Comment (SoonIter) [reply website/theme/components/Landingpage/Background.compressed.png:1]: Does svg also need to be moved?

Comment (chenjiahan) [reply website/theme/components/Landingpage/Background.compressed.png:1]: SVG is small and can be preserved in this repo

**Comment by chenjiahan** [comment]
> I think rspress theme is a better choice because it is a homepage and it can be used like

Some sites may need to customize the page or theme, so providing components makes it easier to compose.

File: website/theme/components/Landingpage/Background.compressed.png
Comment (SoonIter) [reply website/theme/components/Landingpage/Background.compressed.png:1]: can be merged first, then extract shared component with 
https://github.com/rspack-contrib/rstack-design-resources/pull/8

Comment (chenjiahan) [inline website/theme/components/Landingpage/Background.compressed.png:1]: delete?

**Review by chenjiahan — CHANGES_REQUESTED** [review]
The theme variables seems to be broken:

image

File: website/theme/components/Landingpage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react';
+import BackgroundUrl from './Background.compressed.png';
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/index.tsx:2]: unused variable `BackgroundUrl`

**Comment by SoonIter** [comment]
> The theme variables seems to be broken:
> 

It is okay on my machine :D

image

**Comment by SoonIter** [comment]
> The theme variables seems to be broken:


I think it is related to css order and not my fault, lol

image

image

**Comment by chenjiahan** [comment]
Get, I can not reproduce this theme issue too..

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**